src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaCalculator {
```

Comment:
```suggestion
public class Circle extends Figure {
```
це теж можна в Figure винести